### PR TITLE
made changes

### PR DESCRIPTION
**Pull request by someone968**
I have made the necessary and exciting changes and now you can look into it and add or reject as per your requirements

### CODE REVIEW

**Review by shubham2020 — COMMENTED** [review]
i noticed and these changes are pretty impressive

**Review by shubham2020 — CHANGES_REQUESTED** [review]
If only you could add one more function for multiplication that would be like awesome bro. So i hope i see that soon

File: tutorial.py [hunk 1]
```diff
@@ -8,11 +8,15 @@ def sing_song():
     j = 30
     k = i + j
     print(k)
-
+    
+def lovely_changes():
+    print("Excited to see its result")
+    
 def print_song(input_is):
     print("What is a input {}".format(input_is))
```
Comment (someone968) [inline tutorial.py:17]: It is not necessary as it is a cosmetic feature so we can kind of take it ahead without further changes and hence thanks for the suggestion

Comment (shubham2020) [reply tutorial.py:17]: Thank you for your reply i ll be merging this code asap.

**Review by shubham2020 — APPROVED** [review]

**Pull request merged**